README.md

```diff
@@ -1 +1,24 @@
-# hw01
\ No newline at end of file
+# About Me
+Hello, my name is Tianyi Qian.  
+
+### Educational Background  
+
+---
+
+I'm a grad student in the MAPSS program at the University of Chicago and I concentrate in **Quantitative Methods and Social Anlaysis**. I studied **Sociology** and **Media Studies** as an undergrad. I'm hoping to learn the following from this class:
+
+
+* _data manipulation_
+* _data visualization_
```

Comment:
Merit: Great job integrates both lists and italics to demonstrate your goals for this course in an ordered and clear pattern!